### PR TITLE
Minor formatting updates

### PR DESCRIPTION
**Pull request by eliotsaylor**
These tutorials are fantastic! Just wanted to contribute some minor formatting fixes, I don't think the typescript directives are needed in the viewer.js. 
Currently working on adding the units to the model summary tab but need to figure out why the forge api is not returning display units for model properties

### CODE REVIEW

**Comment by eliotsaylor** [comment]
Added a toDisplayUnits() function which turns the units function from "autodesk.autodesk.unit-versionnumber" to "value unit" for the model summary tutorial

File: docs/tutorials/01-simple-viewer/04-viewer.mdx [hunk 1]
```diff
@@ -18,8 +18,6 @@ Let's start by implementing the Forge Viewer functionality for our application.
 Create a `viewer.js` file under the `wwwroot` subfolder with the following code:
 
 ```js title="wwwroot/viewer.js"
-/// import * as Autodesk from "@types/forge-viewer";
```
Comment (petrbroz) [inline docs/tutorials/01-simple-viewer/04-viewer.mdx]: What's the reason for removing the directive? We include it in order to provide static typing for the viewer classes (https://twitter.com/ipetrbroz/status/1327240538697961472). I do agree though that we should probably clarify this in the tutorial (incl. the fact that it only works for Node.js projects when you install the `@types/forge-viewer` dependency. How about adding this to a `:::tip` note?

**Review by petrbroz — APPROVED** [review]

File: docs/tutorials/01-simple-viewer/04-viewer.mdx (hunk 1 above)
Comment (eliotsaylor) [reply docs/tutorials/01-simple-viewer/04-viewer.mdx]: Hey Petr, shoot, my bad! I thought that line was left in there erroneously and didn't realize we had Typescript support. Will fix that and add a note!

**Comment by eliotsaylor** [comment]
Added the TypeScript import back into the viewer in both the Simple Viewer tutorial and the Hubs Browser tutorial! Let me know how that looks 👀

**Review by petrbroz — APPROVED** [review]

**Comment by petrbroz** [comment]
Looks good, thanks @eliotwinchell!

**Pull request merged**